libraries/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -690,4 +702,23 @@ export class ChangeAction extends BaseRushAction {
   private _logNoChangeFileRequired(): void {
     console.log('No changes were detected to relevant packages on this branch. Nothing to do.');
   }
+
+  private _stageGitChanges(patterns: string[]): void {
```

Comment:
Why does it receive an array as parameter and not a single `string`?

Reply:
Moved to just a single string